maven/bnd-resolver-maven-plugin/pom.xml

```diff
@@ -84,7 +88,7 @@
                       <jsse.enableSNIExtension>false</jsse.enableSNIExtension>
                     </properties>
                     <goals>
-                        <goal>bnd-resolver:resolve</goal>
+                        <goal>package</goal>
```

Comment:
i.e. the top level reactor pom doesn't have the plugin configuration for the resolver. Only the children do.